src/main/kotlin/org/elm/lang/core/types/TypeRendering.kt

```diff
@@ -55,3 +55,25 @@ private fun StringBuilder.renderLink(refText: String, text: String, linkify: Boo
     if (linkify) DocumentationManagerUtil.createHyperlink(this, refText, text, true)
     else append(text)
 }
+
+
+fun Ty.renderParam(): String {
+    return alias?.name?.toFirstCharLowerCased() ?: when (this) {
+        is TyFunction -> "function"
+        is TyUnion -> name.toFirstCharLowerCased()
+        is TyRecord -> "record"
+        is TyTuple -> renderParam()
+        is TyVar -> name
+        is TyUnit -> "()"
+        is TyUnknown, TyInProgressBinding -> "unknown"
+        TyShader -> "shader"
+    }
+}
+
+fun TyTuple.renderParam(): String {
+    return types.joinToString(", ", prefix = "(", postfix = ")") { it.renderParam() }
```

Comment:
I chose to destructure tuples in params since that seems pretty common, but I'm happy to just render `tuple` if you think we should be consistent with the other types.